namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java

```diff
@@ -88,51 +89,51 @@ public RemotingCommand processRequest(ChannelHandlerContext ctx,
 
         switch (request.getCode()) {
             case RequestCode.PUT_KV_CONFIG:
-                return this.putKVConfig(ctx, request);
```

Comment:
IMO, It may be better to use a uniform signature for these methods, the contents here may be used later.

Reply:
but five years have passed since last modification
![image](https://user-images.githubusercontent.com/16064908/179887642-085f7e67-6d16-4d3f-b99e-a28bd7658e03.png)
![image](https://user-images.githubusercontent.com/16064908/179887667-7870320f-971c-4f6e-93f8-482bb88d7d94.png)

Reply:
> IMO, It may be better to use a uniform signature for these methods, the contents here may be also used later.

file rolled back